SUMMARY.md

```diff
@@ -24,6 +24,10 @@
 * [Tags and placeholders](applications/advanced-concepts/tags-and-placeholders.md)
 * [Multi-engine machine translation](applications/advanced-concepts/multi-engine-machine-translation.md)
 
+## Integrations
+
+* [How to connect CAT tools to machine translation APIs](api_mt/index.md)
```

Comment:
Will this link actually be valid?

Note that we do not use underscore in urls.

I think the right structure would be something like:

- `integration`
  - `api-keys.md`
  - `cat-tools.md` (currently `index.md`)
  - `memoq.md`

Reply:
I changed structure to what @bittlingmayer proposed.